packages/frontend/src/utils/getWalletURL.js

```diff
@@ -36,4 +36,11 @@ export const getMyNearWalletUrlFromNEARORG = (https = true) => {
     return `${https ? 'https://' : ''}${prefix || ''}mynearwallet.com`;
 };
 
+export const getMeteorWalletUrl = () => {
+    const locallySetBaseUrl =
```

Comment:
Due to security reasons, lets not retrieve the base URL from local storage. I can see a potential attack vector with this for users potentially sending them to a fake site. Can you please just hard code it to be `https://wallet.meteorwallet.app` instead?